src/edge_op.cpp

```diff
@@ -567,7 +570,7 @@ void Manifold::Impl::CollapseEdge(const int edge, std::vector<int>& edges) {
   const vec3 pNew = vertPos_[endVert];
   const vec3 pOld = vertPos_[toRemove.startVert];
   const vec3 delta = pNew - pOld;
-  const bool shortEdge = la::dot(delta, delta) < tolerance_ * tolerance_;
+  const bool shortEdge = la::dot(delta, delta) < epsilon_ * epsilon_;
```

Comment:
Should this be epsilon? I thought we want to remove edges smaller than tolerance (tolerance is the shortest edges that the user want).

Reply:
No, actually the check was already changed to epsilon above, but this line was forgotten. Really the flagged edge collapse should be a superset of the short edge collapse, but the actually the short edges relax a few things. I think that's really only valid for degenerate triangles, rather than "within tolerance" - all the longer edges should really go through the additional flagged edge checks.